<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslateView</class>
 <widget class="QWidget" name="TranslateView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="translationTable">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="autoTranslateAllButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Auto Translate All</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ExpendedEdit" name="expendedEdit" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpendedEdit</class>
   <extends>QWidget</extends>
   <header>Ui/TranslateView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>autoTranslateAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>translateAllSlot()</slot>
  <slot>translateRowSlot(int)</slot>
 </slots>
</ui>
